packages/core/src/browser/preferences/preference-service.ts

```diff
@@ -27,11 +27,31 @@ import { PreferenceConfigurations } from './preference-configurations';
 
 export { PreferenceScope };
 
+/**
+ * Representation of a preference change. A preference value can be set to `undefined` for a specific scope.
+ * This means that the value from a more general scope will be used.
+ */
 export interface PreferenceChange {
+    /**
+     * The name of the changed preference.
+     */
     readonly preferenceName: string;
+    /**
+     * The new value of the changed preference.
+     */
     readonly newValue?: any;
+    /**
+     * The old value of the changed preference.
+     */
     readonly oldValue?: any;
+    /**
+     * The {@link PreferenceScope} of the changed preference.
+     */
     readonly scope: PreferenceScope;
+    /**
+     * Tests wether the given resource is affected by the preference change.
+     * @param resourceUri The uri of the resource to test.
```

Comment:
```suggestion
     * @param resourceUri the uri of the resource to test.
```

For consistency with the rest of the pull-request.